src/invoices/InvoiceLineDetails/InvoiceLineDetails.js

```diff
@@ -21,14 +25,21 @@ import {
   expandAll,
   toggleSection,
 } from '../../common/utils';
+import { configTags } from '../../common/resources';
 import AdjustmentsDetails from '../AdjustmentsDetails';
 
 class InvoiceLineDetails extends Component {
+  static manifest = Object.freeze({
```

Comment:
It makes things messy. You have a container component , InvoiceLineDetailsLayer, that is connected to stripes already. InvoiceLineDetails is supposed to be a View(presentation). I think you can leverage tags into a separate component, connected to backend(stripes). Or as a last resort you can fetch tags in a container component.

Reply:
Created separate component <TagBadge />